ddtrace/opentelemetry/span.go

```diff
@@ -45,6 +49,40 @@ func (s *span) SetName(name string) {
 	s.attributes[ext.SpanName] = strings.ToLower(name)
 }
 
+func stringifySpanEvents(evts []otelsdk.Event) (s string) {
+	for i, e := range evts {
+		if i == 0 {
+			s += marshalSpanEvent(e)
+		} else {
+			s += "," + marshalSpanEvent(e)
+		}
+	}
+	return s
+}
+
+func marshalSpanEvent(evt otelsdk.Event) string {
+	type ddSpanEvent struct {
+		Name           string                 `json:"name"`
+		Time_unix_nano int64                  `json:"time_unix_nano"`
+		Attributes     map[string]interface{} `json:"attributes,omitempty"`
+	}
+	spEvt := ddSpanEvent{
+		Time_unix_nano: evt.Time.Unix(),
+		Name:           evt.Name,
+	}
+	spEvt.Attributes = make(map[string]interface{})
+	for _, a := range evt.Attributes {
+		spEvt.Attributes[string(a.Key)] = a.Value.AsInterface()
+	}
+	s, err := json.Marshal(spEvt)
+	if err != nil {
+		// log something?
+		log.Debug(fmt.Sprintf("Issue marshaling span event %v:%v", spEvt, err))
+		return ""
```

Comment:
what to do with this error? Should we keep the log there?

Reply:
@mtoffl01 I think it's fine like this.